framework/src/utils/InputParameters.C

```diff
@@ -1143,6 +1143,20 @@ InputParameters::addRequiredParam<std::vector<MooseEnum>>(
   metadata._doc_string = doc_string;
 }
 
+template <>
+void
+InputParameters::addRequiredParam<std::vector<MultiMooseEnum>>(
+    const std::string & name,
+    const std::vector<MultiMooseEnum> & moose_enums,
+    const std::string & doc_string)
+{
+  InputParameters::set<std::vector<MultiMooseEnum>>(name) =
+      moose_enums; // valid parameter is set by set_attributes
```

Comment:
I think we should check (assert) that moose_enums is
- not empty
- size 1 (only one enum is supported later in the parsing, we dont support a parameter like {MultiMooseEnum(a b c), MultiMooseEnum(d e)}

Reply:
I've added checks for those